GitHubExplorer/GitHubExplorer/Utils/WithOptionalUILabel.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  WithOptionalTextLabel.swift
+//  GitHubExplorer
+//
+//  Created by ts51 on 12.07.20.
+//  Copyright © 2020 example. All rights reserved.
+//
+
+import UIKit
+
+// Extracted if used later for other optional labels
+protocol WithOptionalUILabel {
+    
+}
+
+extension WithOptionalUILabel {
+    func renderOptionalLabelText<T>(label: UILabel, textValue: T?, prefix: String?, renderPredicate: (T?) -> Bool = { value in
```

Comment:
This make very little sense as an empty protocol with a default function. Better extract to a utility class, ad an explicit dependency on the utility class and use as an instance function of the utility.